src/core/map/popover/MapPopover.fixture.tsx

```
@@ -0,0 +1,430 @@
+      <h4>Enhanced Features: Custom Style + Error Handling</h4>
+      <div
+        ref={mapRef}
+        style={{
+          width: '100%',
+          height: '60vh',
+          border: '1px solid #ddd',
+          borderRadius: '8px',
+        }}
+      />
+      <div style={{ position: 'absolute', top: 35, right: 10, zIndex: 1 }}>
+        <button onClick={close} style={{ background: '#10b981' }}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)